### PR TITLE
BIR-8: Change Difficulty type from int to enum

### PR DESCRIPTION
**Pull request by martamrp**

### CODE REVIEW

File: src/Server/Repositories/QuestionRepository.cs [hunk 1]
```diff
@@ -13,6 +13,7 @@ public class QuestionRepository : IQuestionRepository
     {
         private readonly Random _random = new();
         private readonly IList<Question> _questions = new List<Question>
+
```
Comment (adrianrnieto) [inline src/Server/Repositories/QuestionRepository.cs]:
```suggestion
```

File: src/Server/Repositories/QuestionRepository.cs [hunk 2]
```diff
@@ -51,7 +52,9 @@ public class QuestionRepository : IQuestionRepository
                     "Cuervo"
                 },
                 CorrectAnswer = "Estornino",
-                Difficulty = 1
+                Difficulty = Difficulty.low
+
+
```
Comment (adrianrnieto) [inline src/Server/Repositories/QuestionRepository.cs]:
```suggestion
```

File: src/Shared/Difficulty.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using Birds.Shared;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Birds.Shared
+{
+    public enum Difficulty
```
Comment (adrianrnieto) [inline src/Shared/Difficulty.cs]: Rename this enum to `Difficulties`. By default, enums must be in plural

File: src/Shared/Difficulty.cs [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    {
+        low = 0,
+        medium = 1,
+        high = 2
```
Comment (adrianrnieto) [inline src/Shared/Difficulty.cs]: It is a .NET standard to have the names in pascal casing, rename these properties accordingly

**Review by adrianrnieto — CHANGES_REQUESTED** [review]

File: src/Shared/Difficulty.cs (hunk 3 above)
Comment (martamrp) [reply src/Shared/Difficulty.cs]: Done!

File: src/Shared/Difficulty.cs (hunk 4 above)
Comment (martamrp) [reply src/Shared/Difficulty.cs]: Done

**Review by adrianrnieto — APPROVED** [review]

**Pull request merged**